target/linux/realtek/image/rtl930x.mk

```diff
@@ -131,4 +130,16 @@ define Device/zyxel_xgs1250-12-a1
 	zyxel-vers | \
 	uImage gzip
 endef
```

Comment:
Maybe give lzma & uimage lzma a chance to save 1MB of space. Bootloader is capable of that but I do not know if extraction overwrites itself because of low source address.

Reply:
I would rather not make kind of change right now. I can maybe try separately, or someone else. The 1210 devices also use gzip.